Makefile

```diff
@@ -165,11 +140,34 @@ docker:
 	docker build -t "$(DOCKER_TAG)" --build-arg=GIT_VERSION=$(GIT_VERSION) --build-arg=BUILD_TARGET=$(BUILD_TARGET) .
 
 .PHONY: docker-local
-docker-local:
-	tar ch ./bin Dockerfile.local | docker build -f Dockerfile.local -t "$(DOCKER_TAG)" --build-arg=GIT_VERSION=$(GIT_VERSION) --build-arg=BUILD_TARGET=$(BUILD_TARGET) -
+docker-local: scripts
+	tar ch ./bin Dockerfile.local ./scripts ./config | docker build -f Dockerfile.local -t "$(DOCKER_TAG)" --build-arg=GIT_VERSION=$(GIT_VERSION) --build-arg=BUILD_TARGET=$(BUILD_TARGET) -
 
+.PHONY: test
 test: docker
 	go test ./test/e2e/*.go -v --logtostderr
 
+.PHONY: test-local
 test-local: docker-local
 	go test ./test/e2e/*.go -v --logtostderr
+
+.PHONY: scripts
+scripts:
+	cp -Rv ./scripts/* ./bin
+
+.PHONY: box-dev
+box-dev: scripts
+	exec docker run \
+	-v $$HOME/code/livepeer-in-a-box-database-snapshots:/data \
+	-v $$(realpath bin):/usr/local/bin \
+	-v $$(realpath ../studio):/studio \
+	-v $$(realpath config):/config:ro \
+	--rm \
+	-it \
+	--name catalyst \
+	--shm-size=4gb \
+	-p 1935:1935 \
+	-p 8888:8888 \
+	-p 4242:4242 \
+	-e ATHIEST=true livepeer/catalyst:latest MistController \
+	-c /config/full-stack.json
```

Comment:
Maybe `box-dev` could be changed into a `docker-compose` service or coexist... like we have `dev-container` in infra repo

Reply:
Maybe! Having RabbitMQ and CockroachDB as separate dependencies is certainly something I'm looking at to speed up the startup time. That said, I've always found docker-compose to be kind of a mediocre dev environment and I've never found myself using it for daily development more than a week or so. The cool thing about dev in a single container is that container can be seamlessly deployed to test with others!

But yeah, I might have to live with separate database/queue stuff for the sake of a quicker startup time. Rabbit takes soooo long.

Reply:
I don't have that bad experience with docker-compose but I never used alternatives like `minikube`.